docs/site/Migrating-from-LoopBack-3.md

```diff
@@ -0,0 +1,160 @@
+---
+lang: en
+title: 'Migrating from LoopBack 3'
+keywords: LoopBack 4.0, LoopBack 4, LoopBack 3
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Migrating-from-LoopBack-3.html
+---
+
+For current LoopBack 3 users who want to migrate to LoopBack 4, LoopBack 4
+offers a way to mount your LoopBack 3 application in a LoopBack 4 project. By
+adding your application this way, the application's REST API is included in the
+OpenAPI spec provided by the LoopBack 4 application. This also means that the
+LoopBack 3 application's models can be used with the LoopBack 4 REST API
+Explorer.
+
+## Mounting the LoopBack 3 Application in a LoopBack 4 Project
+
+{% include note.html content="
+If you are not familiar with LoopBack 4, visit
+[Getting started](Getting-started.md) for an introduction.
+" %}
+
+1. To mount your LoopBack 3 application, you will need to first create a new
```

Comment:
The following part is the same as https://github.com/strongloop/loopback-next/blob/master/examples/lb3-application/README.md#tutorial, so I'm wondering if I should just link to there instead or keep it here?

Reply:
I would suggest avoid copy paste duplicate content so that everytime we only update one place, and other places link to the source page.

Reply:
That's a good point! I'll link then, thank you 👍